src/inputs/radio/radio.jsx

```diff
@@ -31,6 +46,9 @@ const propTypes = {
         labelNotClickable: PropTypes.string,
         root: PropTypes.string,
     }),
+    /**
+     * Additional classes.
```

Comment:
Change comment to: "Assign additional class names to Radio."

Reply:
done!